cmd/gitops/ui/run/cmd.go

```diff
@@ -78,6 +83,11 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().StringVar(&options.NotificationControllerAddress, "notification-controller-address", "", "the address of the notification-controller running in the cluster")
 	cmd.Flags().IntVar(&options.WatcherPort, "watcher-port", 9443, "the port on which the watcher is running")
 
+	cmd.Flags().BoolVar(&options.Insecure, "insecure", false, "allow insecure TLS requests")
+	cmd.Flags().StringVar(&options.TLSCert, "tls-cert", "", "filename for the TLS certficate, in-memory generated if omitted")
```

Comment:
We could reuse the same naming scheme as kube-apiserver here... `--tls-cert-file` and `--tls-private-key-file` ?

Reply:
sgtm!